frontend/templates/product-list/sections/FilterableProductsSection/useSortBy.ts

```diff
@@ -63,6 +63,6 @@ export function useSortBy(
       case 'facet_tags.Item Type':
          return sortAlphabetically;
       default:
-         return props.sortBy;
+         return ["count:desc", "name:asc"];
```

Comment:
I think this approach makes sense. According to the [docs](https://www.algolia.com/doc/api-reference/widgets/refinement-list/react-hooks/?client=Function+%28TypeScript%29#hook-params-sortby), it looks like we can also update the default ordering when we [configure](https://www.algolia.com/doc/api-reference/api-parameters/renderingContent/?client=php) the index in PHP, but the intent is a little more explicit this way.

Reply:
I like us defining it explicitly here, considering the rest of the sorts are also defined in this file